Build started 27.06.2018 16:46:29.
     1>Project "C:\Users\alba940\Downloads\PythonQt3.2\examples\PyCPPWrapperExample\PyCPPWrapperExample.vcxproj" on node 8 (Build target(s)).
     1>CustomBuild:
         MOC CustomObjects.h
         RCC PyCPPWrapperExample.qrc
       ClCompile:
         C:\Program Files (x86)\Microsoft Visual Studio 12.0\VC\bin\x86_amd64\CL.exe /c /I. /I..\..\src /ID:\Qt\5.7\msvc2013_64\include /ID:\Qt\5.7\msvc2013_64\include\QtWidgets /ID:\Qt\5.7\msvc2013_64\include\QtGui /ID:\Qt\5.7\msvc2013_64\include\QtANGLE /ID:\Qt\5.7\msvc2013_64\include\QtCore /Idebug /I"D:\Qt\5.7\msvc2013_64\mkspecs\win32-msvc2013" /Zi /nologo /W3 /WX- /Od /D _WINDOWS /D UNICODE /D WIN32 /D WIN64 /D QT_WIDGETS_LIB /D QT_GUI_LIB /D QT_CORE_LIB /Gm- /EHsc /MDd /GS /fp:precise /Zc:wchar_t /Zc:forScope /GR /Fo"debug\\" /Fd"debug\vc120.pdb" /Gd /TP /errorReport:prompt -w34100 -w34189 -w44996 CustomObjects.cpp main.cpp debug\moc_CustomObjects.cpp debug\qrc_PyCPPWrapperExample.cpp
         CustomObjects.cpp
     1>c:\users\alba940\downloads\pythonqt3.2\src\PythonQtPythonInclude.h(59): fatal error C1083: Cannot open include file: 'Python.h': No such file or directory
         main.cpp
     1>c:\users\alba940\downloads\pythonqt3.2\src\PythonQtPythonInclude.h(59): fatal error C1083: Cannot open include file: 'Python.h': No such file or directory
         moc_CustomObjects.cpp
     1>c:\users\alba940\downloads\pythonqt3.2\src\PythonQtPythonInclude.h(59): fatal error C1083: Cannot open include file: 'Python.h': No such file or directory
         qrc_PyCPPWrapperExample.cpp
         Generating Code...
     1>Done Building Project "C:\Users\alba940\Downloads\PythonQt3.2\examples\PyCPPWrapperExample\PyCPPWrapperExample.vcxproj" (Build target(s)) -- FAILED.

Build FAILED.

Time Elapsed 00:00:01.18
